src/App2.js

```diff
@@ -0,0 +1,29 @@
+import logo from './logo.svg';
+import './App.css';
+import React from 'react';
+import { Component } from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/shekhar-rudder/code-code/pullRequest?prid=7908803)
```suggestion
import { Component } from "react";
```